docs/plugins/ingest-attachment.asciidoc

```diff
@@ -238,3 +238,67 @@ properties on this top level field will contain the value of the
 first attachment only. However, by specifying the
 `target_field` on to a value on `_ingest._value` it will correctly
 associate the properties with the correct attachment.
+
+[[ingest-attachment-raw-metadata]]
+==== Extracting all properties from documents
+
+By default, the `ingest-attachment` plugin only extracts a subset of the most common properties:
+
+* `content`: the extracted text
```

Comment:
the whole notion/trick with the underscores is not mentioned in the docs... am I misreading something?

Reply:
Marking as solved as I'm not using that anymore.